.travis.yml

```diff
@@ -1,12 +1,41 @@
+conditions: v1
 language: python
+
 python:
-  - "2.6"
-  - "2.7"
-  - "3.2"
-  - "3.3"
-  - "3.4"
-  - "3.5"
-  - "3.5-dev" # 3.5 development branch
-  - "nightly" # currently points to 3.6-dev
-# command to run tests
+- 2.6
+- 2.7
+- 3.2
+- 3.3
+- 3.4
+- 3.5
+- 3.6
+- 3.7
```

Comment:
It's already in `jobs.include` + it doesn't work like others and will error out in Travis CI.
```suggestion
```